fiat-plugins/roles-file/roles-file.gradle

```diff
@@ -15,15 +15,10 @@
  */
 
 dependencies {
-  implementation project(":fiat-roles")
-  implementation project(":fiat-core")
-  implementation "com.fasterxml.jackson.core:jackson-core"
-  implementation "com.fasterxml.jackson.core:jackson-databind"
+  implementation project(":fiat-main")
```

Comment:
I think I could revert the dependency changes in here

I initially was looking at bringing in fiat-main as a `compileOnly` dependency, but with the dependency shenanigans I added that is no longer necessary